docs/system-admin-guide/users-permissions/roles-permissions/README.md

```diff
@@ -93,6 +93,17 @@ OpenProject allows to share project information with **anonymous** users which a
 | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
 | Project-level: Permissions scoped to individual projects for users which are <u>not</u> logged in | - View work packages for users that are not logged in        | Assign different permissions to the role *Anonymous*         |
 
+### Standard
+
+**Standard** is the default role of users of your OpenProject instance. It is configured and assigned by administrators on an instance level. <br>
```

Comment:
Sorry, in this case the assignment is not true, because it is automatically assigned.

```suggestion
**Standard** is the default role of users of your OpenProject instance. It is configured by administrators on the instance level. <br>
```